src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -929,6 +932,9 @@
   - name: --vnet-subnet-id
     type: string
     short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
+  - name: --pod-subnet-id
```

Comment:
Hi [mattstam](https://github.com/mattstam),
I can only find `aks nodepool add --pod-subnet-id` in _help.py file.
Can you please take a look ?

Reply:
On Line 384 it is also present for `aks create --pod-subnet-id`, what is missing?

Reply:
Sorry, I didn't make myself clear.
Why not also add `c.argument('pod_subnet_id', type=str, validator=validate_pod_subnet_id)` in `aks nodepool add` in _params.py file ？

Reply:
Added in https://github.com/Azure/azure-cli/pull/21651/commits/e0ad6769770a1f872b7a97b9129b2b34c9bfbba2, thanks for the suggestion